ceph_bootstrap/config_shell.py

```diff
@@ -308,6 +308,27 @@ def possible_values(self):
             },
         }
     },
+    'System_Upgrade': {
```

Comment:
For consistency, let's use `Update` instead of `Upgrade` (rename all occurrences)

Reply:
Done